K8s-deployment/K8s-cluster/addons/storage/azure/README.md

```diff
@@ -43,13 +43,55 @@ kubectl create -f azure-cloud-provider.yaml
 ``` kubectl apply -f azuredisk-storage-class.yaml```
 
 # Deployment with AzureFile
-1. Create a storage account on azure for the shared file system. Follow the documentation [here](https://docs.microsoft.com/en-us/azure/storage/common/storage-account-create?tabs=azure-portal).
-2. Update the `azurefile-storageclass.yaml` with the created storage account name.
-3. Following command will deploy the storageclass
+For in-tree azuredisk drivers, ```kubernetes.io/azure-file``` provisioner can be used in the [azurefile-storage-class.yaml](./azurefile-storage-class.yaml).
```

Comment:
Document azurefile csi drivers
- Minimum k8s version - 1.21+
- Pre-requisites
- Installation of csi drivers through helm chart with commands 
and ref: https://github.com/kubernetes-sigs/azurefile-csi-driver#install-driver-on-a-kubernetes-cluster